DoSomething/quasar
# Updating Pipfile to install dbt-postgresql vs dbt to fix boto conflicts

## What's this PR do?
 I got some more `boto` version conflicts when checking out the latest the version of the Quasar repo and running `pipenv install`. Changing our Pipfile to only use the `dbt-postgres` package appears to fix this.
#### Where should the reviewer start?
https://github.com/DoSomething/quasar/compare/dbt-postgres?expand=1#diff-1e61a31bf9b94805f869dc4137ec1885R31
#### How should this be manually tested?
Checkout current Quasar master branch and try to install `pipenv install`. You'll should get the `boto` conflicts. 

Then checkout this branch to a new directory and try `pipenv install`, should be conflict free.
#### Any background context you want to provide?
#### What are the relevant tickets?
Related to [this](https://www.pivotaltracker.com/story/show/170649560) ticket.
